session/session.go

```diff
@@ -3510,7 +3519,7 @@ func BootstrapSession(store kv.Storage) (*domain.Domain, error) {
 		dom.Close()
 		return nil, err
 	}
-
+	dom.InitInfo4Test()
```

Comment:
If  we put it to the end of `BootstrapSession`, could we put it to `BootstrapSession4DistExecution`?And then we will not affect the normal code.

Reply:
> If we put it to the end of `BootstrapSession`, could we put it to `BootstrapSession4DistExecution`?And then we will not affect the normal code.

Unluckily `InitDistTaskLoop` in ut rely on InitInfo4Test to generate mock serverID.
I will put line 3552 into `InitDistTaskLoop` line 1384